modules/swagger/src/main/scala/ru/tinkoff/tschema/swagger/SwaggerType.scala

```diff
@@ -285,9 +284,12 @@ object SwaggerType {
               case (name, None, obj)        => name -> obj.asJson
               case (name, Some(descr), obj) => name -> obj.add("description", Json.fromString(descr)).asJson
             }
-            JsonObject("type"       -> Json.fromString("object"),
-                       "required"   -> Json.arr(required.value.map(Json.fromString): _*),
-                       "properties" -> Json.obj(fields: _*))
+
+            JsonObject.fromIterable(
+              "type" -> Json.fromString("object") ::
+                required.value.toNev.map("required" -> _.asJson).toList
```

Comment:
Could it be just
```
"required" -> required.value.toNev.fold(Json.Null)(_.asJson)
```
?

Reply:
Yep, since we already are putting nulls for `Skippable` values.